contracts/lbp/LBPWrapper.sol

```diff
@@ -18,11 +18,14 @@ import "../utils/interface/IVault.sol";
 import "../utils/interface/ILBP.sol";
 
 contract LBPWrapper {
+    uint256 public constant SWAP_FEE_PERCENTAGE = 1e12; // 0.0001% is minimum amount.
+
     address public owner;
-    ILBPFactory public LBPFactory;
-    address public lbp;
-    uint256 public swapFeePercentage;
-    bool isInitialized;
+    bool public isPoolFunded;
+    bool public isInitialized;
+    bool public isPaused;
```

Comment:
Not sure if we actually need this additional state variable. I believe we can get the information on whether the pool is paused or not from [here](https://github.com/balancer-labs/balancer-v2-monorepo/blob/master/pkg/pool-weighted/contracts/smart/LiquidityBootstrappingPool.sol#L137).

Reply:
I think you mean the `isPaused` variable right? I thought it would be more expensive to do a call to another contract, that's why I decided to have our own variable. I could implement a function instead like `getPausedEnabled()`?

Reply:
Intuitively I would say that implementing a function to read the status is cheaper than writing and storing additional data. I would follow your suggestion to implement a getter function, call it isPaused() maybe?